lib/scaffold.dart

```diff
@@ -600,10 +603,13 @@ class BackdropScaffoldState extends State<BackdropScaffold>
               children: <Widget>[
                 // subHeader
                 _MeasureSize(
-                  onChange: (size) =>
-                      setState(() => _subHeaderHeight = size.height),
+                  onChange: (size) {
+                    if (size != null) {
+                      setState(() => _subHeaderHeight = size.height);
+                    }
+                  },
```

Comment:
Added enclosing `size != null` check. Is this ok?

Reply:
Resolved in https://github.com/fluttercommunity/backdrop/commit/2f4010e5619ad2089290835c3e40b7048594943f.